xmtp-inbox-ios/Loaders/MessageLoader.swift

```diff
@@ -34,20 +34,25 @@ class MessageLoader: ObservableObject {
 		}
 	}
 
+	func streamTopic(topic: DB.ConversationTopic) async {
+		do {
```

Comment:
In case a message gets streamed in that we already have and there's a race on the xmtpID uniqueness, this lets us retry.